eng/pipelines/official.yml

```diff
@@ -169,3 +169,7 @@ extends:
         desiredFinalVersionKind: ${{ parameters.desiredFinalVersionKind }}
         isOfficialBuild: true
         scope: full
+        # Exclude runtime dependent jobs for any branch not producing a 1xx version since runtime-related repos aren't built in that context
+        # Will exclude runtime-dependent jobs when the branch is not main and does not contain '.1xx' but does end with 'xx'.
+        ${{ if and(ne(variables['Build.SourceBranch'], 'refs/heads/main'), not(contains(variables['Build.SourceBranch'], '.1xx')), endsWith(variables['Build.SourceBranch'], 'xx')) }}:
```

Comment:
I don't think I'd use an expression for this. A dev build of an upstack component (or hotfix branches) might have different branch names or format. Since 2xx is branched from 1xx, then 3xx from 2xx, etc., I would just enable this in 2xx when we branch.